sdk/quantum/arm-quantum/CHANGELOG.md

```diff
@@ -1,15 +1,33 @@
 # Release History
+    
+## 1.0.0-beta.2 (2024-03-12)
+    
+**Features**
 
-## 1.0.0-beta.2 (Unreleased)
+  - Added operation Workspace.listKeys
+  - Added operation Workspace.regenerateKeys
+  - Added Interface ApiKey
+  - Added Interface APIKeys
+  - Added Interface ListKeysResult
+  - Added Interface WorkspaceListKeysOptionalParams
+  - Added Interface WorkspaceRegenerateKeysOptionalParams
+  - Added Interface WorkspaceResourceProperties
+  - Added Type Alias KeyType_2
+  - Added Type Alias WorkspaceListKeysResponse
+  - Interface QuantumWorkspace has a new optional parameter properties
+  - Interface Resource has a new optional parameter systemData
+  - Added Enum KnownKeyType
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+**Breaking Changes**
```

Comment:
Can you link the spec pr of breaking change?

Reply:
it is because disable flatten